src/containers/list.jsx

```diff
@@ -1,11 +1,56 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import NavBar from './../components/nav_bar.jsx';
 
-const List = () => (
-  <div>
-    <h1>This is list page</h1>
-    <NavBar />
-  </div>
-);
+class List extends Component {
+  constructor(props) {
+    super(props);
 
-export default List;
+    this.state = {};
+
+    this.renderList = this.renderList.bind(this);
+  }
+
+  renderList() {
+    const filtered = [];
+    const listItems = this.props.careers;
+    const likedCareers = JSON.parse(localStorage.liked);
+    listItems.forEach((career) => {
```

Comment:
since this counts as logic **If you want** you can  put it in a different file and give listItems and likedCareers as arguments.
but i feel like it's more self preference here

Reply:
@matthewdking i guess you are going to keep it in the same file?

Reply:
@Karyum as our QA what would you recommend?

Reply:
keep it in the same file i don't think it's going to get used again